bld/setup_linux.sh

```diff
@@ -1,4 +1,10 @@
-#1/bin/sh
+#!/bin/sh
+
+sudo dpkg --add-architecture i386
+sudo apt-get update
+sudo apt-get install linux-libc-dev:i386
+sudo apt-get install gcc-multilib
```

Comment:
Ubuntu 20.04 images didn't come with the packages necessary for x86 builds. I put it along the cross-compilers but it can be moved to the pipeline definition if desired.